buttons/buttons.css

```diff
@@ -1724,3 +1724,23 @@ a:focus-visible {
 }
 /*Nitish's btn end */
 
+/*Thiago's btn start*/
+.Okan-btn {
```

Comment:
its clearly mention in readme.md file that css class should be  like
yourname-button-1 or yourname-button-index
in your case  
Okan-button-1

Reply:
so can u just pls change this thing

Reply:
Ok. I will to fix this.